hayk-hovhannisyan/README.txt

```diff
@@ -0,0 +1,2 @@
+Run with your ssh_key
+terraform apply -var="ssh_key=#####"
```

Comment:
You should consider adding a separate 'variables.tf' file and put variables like ssh key pair name, region, ami-id in it.